src/eve/trees.py

```diff
@@ -55,7 +55,7 @@
     TreeLike = Any
 else:
 
-    class TreeLike(abc.ABC):
+    class TreeLike(abc.ABC):  # noqa: B024
```

Comment:
*looking at the surrounding code*
Hang on, what is happening here? Looks like there must be an interesting story behind all this.

Doesn't concern this PR though

Reply:
Maybe @egparedes wants to tell the story.

Reply:
The think the added `#noqa` is related to an update of flake8 in CI which now triggers this error. This class itself is going to be changed soon, so adding the `noqa` here it's ok for now.